setup.py

```diff
@@ -59,8 +59,6 @@
         'allofplos.seed_corpus': ['*.xml'],
     },
 
-    #include_package_data=True,
```

Comment:
Is this line needed un-commented out?

Reply:
This line is deleted because is not needed. It is used in conjunction with MANIFEST.in file, but this is for previous Python version (less than 3), since we don't support <3, it is not needed.
Here more information http://setuptools.readthedocs.io/en/latest/setuptools.html#including-data-files